### ISSUE TITLE
VGG optimization

### ISSUE THREAD

**Issue by mpapadomanolaki** [wgcban/Metric-CD#4]
Hello,

Thank you for your feedback.
I am trying to understand your method and I have a question.
In your paper you say that the weights of both D-CPG and D-FE are optimized during training.
Could you tell me your approach on this?
Are you using two different optimizers?

Thank you for your time.

Maria

**Comment by wgcban** [comment]
@mpapadomanolaki thanks for your question. In our experiments, we optimize the change probability generator while keeping the VGG feature extractor fixed. But, you can fine-tune the VGG feature extractor as well using a single optimizer.

**Comment by mpapadomanolaki** [comment]
Thank you for your quick answer!
